src/NovelRT/Audio/AudioService.cpp

```diff
@@ -84,7 +85,7 @@ std::vector<ALuint>::iterator AudioService::loadMusic(std::string input) {
 
   //Sorry Matt, nullptr types are incompatible to ALuint according to VS.
   if (newBuffer == _noBuffer) {
-    _logger.logError("Could not load audio file: ", getALError());
+    _logger.logWarning("Could not load audio file: " + input);
```

Comment:
Just wanted to point out here - for some reason the LoggingService iterator does not appear to work. FWIW, `getALError()` was incorrect in this context and was always just producing a blank string, however adding a string such as `_logger.logWarning(string1, string2);` would produce a log message only with `string1`. For now, I resulted to concatenation so that it would work.